packages/@okta/vuepress-theme-prose/const/navbar.const.js

```diff
@@ -44,9 +44,13 @@ export const concepts = [
         path: "/docs/concepts/how-okta-works/"
       },
       {
-        title: "Identity engine",
-        path: "/docs/concepts/ie-intro/",
-        hidden: true
+        title: "Identity Engine",
+        subLinks: [
+          {
+            title: "Redirect vs. embedded authentication",
+            path: "/docs/concepts/redirect-vs-embedded"
+          },
```

Comment:
Just making sure we intend to have a single sub link under the `Identity Engine` section

Reply:
yes - in Concepts - this is all we have for OIE is this one doc.